darwin/Platform.c

```diff
@@ -405,6 +405,7 @@ bool Platform_getDiskIO(DiskIOData* data) {
       return false;
 
    unsigned long long int read_sum = 0, write_sum = 0, timeSpend_sum = 0;
+   unsigned long long int numDisks = 0;
```

Comment:
Possible cleanup: Be consistent with later usage of this variable and use `uint64_t` here too …

Reply:
I meant both lines (just as marked ;-)), as `*_sum` maps to `total*`, which all are `uint64_t` …

Reply:
That one is better done in a separate commit.